types/alloy/index.d.ts

```diff
@@ -148,7 +145,7 @@ interface AlloyInterface {
    * @param name Name of controller to instantiate.
    * @param args Arguments to pass to the controller
    */
-  createController(name: string, args?: any): AlloyController;
+  createController(name: string, args?: any): AlloyController & any;
```

Comment:
`AlloyController & any` = `any` .. Either use specific type or `any` if its suppose to be anything.

Reply:
Refer to https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes

Reply:
Fixed! Sorry about that. That was a brainfart.